desktop/app/store-helper.js

```diff
@@ -1,10 +1,48 @@
 import {ipcMain} from 'electron'
+import {selector as trackerSelector} from '../../react-native/react/tracker'
+import {selector as menubarSelector} from '../../react-native/react/menubar'
 
-export default function () {
-  // In case the subscribe store comes before the remote store is ready
-  ipcMain.on('subscribeStore', event => {
-    ipcMain.on('remoteStoreReady', () => {
-      event.sender.send('resubscribeStore')
+export default function (mainWindow) {
+  const subscribeStoreSubscribers = []
+  let store = {}
+
+  ipcMain.on('subscribeStore', (event, component, selectorParams) => {
+    let selector = {
```

Comment:
We allow a per-component-type filter for sending sub parts of the store. This can cause issues if you're sending parts of the store and forget to update this but it really makes things faster. The implementation is right next to the main connect() function in the components (which does the filtering anyways so maybe this is better). let's discuss